js/boaexchange.js

```diff
@@ -87,32 +86,38 @@ module.exports = class boaexchange extends Exchange {
                 ],
             },
             'api': {
-                'v1': {
+                'public': {
                     'get': [
-                        'addresses',
-                        'addresses/{label}',
                         'airdrops',
-                        'balances',
-                        'balances/{label}',
                         'chat',
                         'coins',
                         'coins/{label}',
-                        'coins/{label}/limits',
-                        'deposits',
-                        'deposits/{depositId}',
                         'ledger',
```

Comment:
Alright! We're making very good progress!

I think, the `airdrops` and the `ledger` endpoints both should be also moved to a private section, right?

Reply:
Removed Ledger. Kept airdrops since its a public endpoint